--- main.cc.orig	2009-06-09 12:34:59.000000000 -0400
+++ main.cc	2009-06-09 12:35:25.000000000 -0400
@@ -224,7 +224,7 @@
 // image. (or for a pnm file if debugging).
 // 'exportfile' is the Ocr Results File.
 //
-int main( const int argc, const char * argv[] )
+/*int main( const int argc, const char * argv[] )
   {
   Input_control input_control;
   Control control;
@@ -387,3 +387,4 @@
   if( control.exportfile ) std::fclose( control.exportfile );
   return retval;
   }
+*/
