--- a/configure.ac
+++ b/configure.ac
@@ -189,7 +189,7 @@
 
 AX_PROBE_OPTIONAL_LIBRARY(
 	[cuneiform],
-	[ctiimage.h cttypes.h puma.h lang_def.h mpuma.h compat_defs.h],
+	[cuneiform.h],
 	[auto],
 	[Enable optional support for Cuneiform (OCR system developed by Cognitive technologies), see https://launchpad.net/cuneiform-linux]
 )
@@ -351,12 +351,7 @@
 dnl	LIBS="${LIBS} -lcuneiform -lrcorrkegl -lrfrmt -lrmarker -lrblock -lrneg -lrout -lced -lrpic -lrselstr -lrstuff -lrimage  -lrline -lrshelllines -lrverline -lcimage -lcfio -lcpage -llns32 -lrdib -lsmetric -lexc -lloc32 -lrreccom -lrpstr -lrstr -lcline -lrcutp -lpass2 -lrbal -lrsadd -lleo32 -levn32 -lfon32 -lctb32 -lmsk32 -ldif32 -lcpu32 -lr3532 -lmmx32 -lrling -lrlings -lcstr -lccom -lstd32 -lcfcompat"
 	AX_TRY_LINK([cuneiform], [
 		#include <stddef.h>
-		#include <ctiimage.h>
-		#include <cttypes.h>
-		#include <puma.h>
-		#include <lang_def.h>
-		#include <mpuma.h>
-		#include <compat_defs.h>
+		#include <cuneiform.h>
 	], [
 		PUMA_XOpen(NULL, "");
 		PUMA_XFinalRecognition();
--- a/src/osra_ocr.cpp
+++ b/src/osra_ocr.cpp
@@ -41,12 +41,7 @@
 #include <ocradlib.h>
 
 #ifdef HAVE_CUNEIFORM_LIB
-#include <ctiimage.h>
-#include <cttypes.h>
-#include <puma.h>
-#include <lang_def.h>
-#include <mpuma.h>
-#include <compat_defs.h>
+#include <cuneiform.h>
 #endif
 
 #include "osra.h"
