--- a/src/osra_ocr.cpp
+++ b/src/osra_ocr.cpp
@@ -67,15 +67,15 @@
 {
 #ifdef HAVE_CUNEIFORM_LIB
   // Initialization for Cuneiform library should be called only once. Otherwise it breaks down during deinitialization:
-  int langcode = LANG_ENGLISH;
-  Bool dotmatrix = 0;
-  Bool fax = 0;
-  Bool onecolumn = 1;
+  int langcode = PUMA_LANG_ENGLISH;
+  PumaBool dotmatrix = 0;
+  PumaBool fax = 0;
+  PumaBool onecolumn = 1;
   PUMA_Init(0, 0);
   PUMA_SetImportData(PUMA_Word32_Language, &langcode);
-  PUMA_SetImportData(PUMA_Bool32_DotMatrix, &dotmatrix);
-  PUMA_SetImportData(PUMA_Bool32_Fax100, &fax);
-  PUMA_SetImportData(PUMA_Bool32_OneColumn, &onecolumn);
+  PUMA_SetImportData(PUMA_PumaBool32_DotMatrix, &dotmatrix);
+  PUMA_SetImportData(PUMA_PumaBool32_Fax100, &fax);
+  PUMA_SetImportData(PUMA_PumaBool32_OneColumn, &onecolumn);
 #endif
 
 #ifdef HAVE_TESSERACT_LIB
@@ -176,7 +176,7 @@
   char str[256];
   memset(str, 0, sizeof(str));
 
-  if (!PUMA_XOpen(blob.data(), NULL) || !PUMA_XFinalRecognition() || !PUMA_SaveToMemory(NULL, PUMA_TOTEXT, PUMA_CODE_ASCII, str, sizeof(str) - 1))
+  if (!PUMA_XOpen((void*) blob.data(), NULL) || !PUMA_XFinalRecognition() || !PUMA_SaveToMemory(NULL, PUMA_TOTEXT, PUMA_CODE_ASCII, str, sizeof(str) - 1))
     {
       //if (verbose)
       //  cout << "Cuneiform recognition failed." << endl;
